app/models/enrollment.rb

```diff
@@ -0,0 +1,7 @@
+class Enrollment < ApplicationRecord
+  enum status: { invited: 0, accepted: 1 }
+  # belongs_to :attendee, class_name: "User"
+  # belongs_to :attended_event, class_name: "Event"
```

Comment:
- [ ] Please commented code. You can use git history to check the difference if you need to